tests/tools/test_convert_partition.py

```diff
@@ -4,25 +4,115 @@
 import numpy as np
 import pytest
 
+from convert_partition import _get_unique_invidx
+
 from tools.distpartitioning import utils
 
-from tools.distpartitioning.convert_partition import _get_unique_invidx
+
+@pytest.mark.parametrize(
+    "num_nodes, num_edges, nid_begin, nid_end",
+    [
+        [4000, 40000, 0, 1000],
+        [4000, 40000, 1000, 2000],
+        [4000, 40000, 2000, 3000],
+        [4000, 40000, 3000, 4000],
+        [4000, 100, 0, 1000],
+        [4000, 100, 1000, 2000],
+        [4000, 100, 2000, 3000],
+        [4000, 100, 3000, 4000],
+        [1, 1, 0, 1],
+    ],
+)
+def test_get_unique_invidx_with_numpy(num_nodes, num_edges, nid_begin, nid_end):
```

Comment:
What is the difference between test_get_unique_invidx_with_numpy and test_get_unique_invidx?

Reply:
test_get_unique_invidx_with_numpy - is the one you requested. Comparing the results with numpy's unique results. and test_get_unique_invidx is the test case for testing our work-around solution (here I validate the results from the workaround solution). I like to keep them separated.